modules/cloudflare.py

```
@@ -59,10 +67,14 @@ async def setZoneUnderAttack(self, zone_id: str, mode: bool) -> dict:
-                    url=url,
-                    headers=await self.getHeaders(),
-                    json=data
-            ) as response:
-                return await response.json()
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.patch(
+                        url=url,
+                        headers=await self.getHeaders(),
+                        json=data
+                ) as response:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)